### PR TITLE
Add waitRound to sendAndWait

### PR DESCRIPTION
**Pull request by vuvoth**
Task details: 
https://www.pivotaltracker.com/n/projects/2452320/stories/182759821

## TODO

- [x] update change log

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -21,6 +21,7 @@ Features, Bug Fixes, API Breaking, Deprecated, Infrastructure, Template Updates
 
 - Add `--npm` flag to `algob init` and `algob unbox`. Note: by default we will use `yarn`.
 - Improved `algob/project-dev-script.sh` which is script setting up a local project.
+- Add `waitRounds` params to `sendAndWait` method. waitRounds have default value is 10.
```
Comment (sczembor) [inline CHANGELOG.md:24]:
```suggestion
- Add `waitRounds` params to `sendAndWait` method. waitRounds default value is 10.
```

File: packages/web/src/lib/myalgowallet-mode.ts [hunk 2]
```diff
@@ -111,19 +111,22 @@ export class MyAlgoWalletSession {
 	/**
 	 * Send signed transaction to network and wait for confirmation
 	 * @param rawTxns Signed Transaction(s)
+	 * @param waitRounds number round wait for comfirmed txn - default is 10
```
Comment (sczembor) [inline packages/web/src/lib/myalgowallet-mode.ts:114]:
```suggestion
	 * @param waitRounds number of rounds to wait for transaction to be confirmed - default is 10
```

**Review by sczembor — APPROVED** [review]
Left few comments.

File: packages/web/src/lib/web-mode.ts [hunk 3]
```diff
@@ -31,7 +31,8 @@ export class WebMode {
 	 * @param txId Transaction id
```
Comment (ratik21) [inline packages/web/src/lib/web-mode.ts:31]: could you please add `waitRounds` here as well?

File: packages/web/src/lib/wallectconnect-mode.ts [hunk 4]
```diff
@@ -183,19 +183,22 @@ export class WallectConnectSession {
 	/**
 	 * Send signed transaction to network and wait for confirmation
 	 * @param rawTxns Signed Transaction(s)
+	 * @param waitRounds number round wait for comfirmed txn - default is 10
```
Comment (ratik21) [inline packages/web/src/lib/wallectconnect-mode.ts:186]:
```suggestion
	 * @param waitRounds number of rounds to wait for transaction to be confirmed - default is 10
```

File: CHANGELOG.md (hunk 1 above)
Comment (ratik21) [reply CHANGELOG.md:24]: Please link `sendAndWait` to the API docs

**Review by ratik21 — APPROVED** [review]

File: packages/web/src/lib/web-mode.ts (hunk 3 above)
Comment (vuvoth) [reply packages/web/src/lib/web-mode.ts:31]: Oh! I forgot... Thanks <3

File: CHANGELOG.md (hunk 1 above)
Comment (robert-zaremba) [reply CHANGELOG.md:24]: Also note that it's an optional argument

File: packages/algob/src/lib/constants.ts [hunk 5]
```diff
@@ -3,6 +3,6 @@ import { Algodv2 } from "algosdk";
 export const globalZeroAddress = "AAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAY5HFKQ";
 export const MIN_UINT64 = 0n;
 export const MAX_UINT64 = 0xffffffffffffffffn;
-export const WAIT_ROUNDS = 6; // for transaction confirmation
+export const WAIT_ROUNDS = 10; // for transaction confirmation
```
Comment (robert-zaremba) [inline packages/algob/src/lib/constants.ts:6]: why increasing? I think 6 was long enough. Also in `sendAndWait` documentation note what's the default

Comment (vuvoth) [reply packages/algob/src/lib/constants.ts:6]: Because in algob/web we use WAIT_ROUNDS = 10

Comment (robert-zaremba) [reply packages/algob/src/lib/constants.ts:6]: then let's import the variable from `web` here.

**Review by robert-zaremba — APPROVED** [review]
Lgtm

**Pull request merged**